edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions message-top-bar" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
```

Comment:
Replace this block with:
"You must set up and start the proctoring software before you begin your exam."